GVRf/Framework/jni/gl/gl_texture.h

```diff
@@ -32,7 +32,7 @@ class GLTexture {
         glBindTexture(target, id_);
         glTexParameteri(target, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
         glTexParameteri(target, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
-        glTexParameteri(target, GL_TEXTURE_MIN_FILTER, GL_LINEAR_MIPMAP_LINEAR);
+        glTexParameteri(target, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
```

Comment:
> why was i seeing a null mesh? and why was it a random number of objects that were visible?

That's a really good question. In all my poking around today, I never saw a `null` mesh or a `null` texture, whether with the original `.notify()` code or @guodongrong 's  `.notifyAll()` change.